DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/DelegateGroups/AddCourseToGroupSelectCourse.cshtml

```diff
@@ -0,0 +1,27 @@
+﻿@inject IConfiguration Configuration
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.DelegateGroups
+@using Microsoft.Extensions.Configuration
+@model AddCourseToGroupCoursesViewModel
+
+<link rel="stylesheet" href="@Url.Content("~/css/trackingSystem/delegates/delegateGroups.css")" asp-append-version="true">

```

Comment:
Does this provide any style for this page?

Reply:
Yes, this imports the searchableElements styles which are currently used for at least the card tags (and will be used for more once we implement search etc.)